test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1279,15 +1279,38 @@ class select_transport_rndv : public test_ucp_wireup {
     {
         add_variant_with_value(variants, UCP_FEATURE_TAG, TEST_TAG, "tag");
     }
+
+    bool has_resource(const ucp_test_base::entity *e, const char *tl_name)
+    {
+        int i;
+        std::string tl(tl_name);
+
+        for (i = 0; i < e->ucph()->num_tls; ++i) {
+            if (tl == e->ucph()->tl_rscs[i].tl_rsc.tl_name) {
+                return true;
+            }
+        }
+
+        return false;
+    }
 };
 
+/* This test checks that DC transport is selected for scaled communication,
+ * even if RNDV_PUT is disabled (default behavior) */
```

Comment:
maybe also set UCX_RNDV_SCHEME=get_zcopy?

Reply:
AFAIR it will pass the test even without the fix, no?

Reply:
will check on GPU when RDMZ storage restored

Reply:
was it checked?

Reply:
test pass